risteys_elixir/assets/css/tailwind.css

```diff
@@ -311,6 +312,13 @@ header .actions a {
     @apply underline
 }
 
+.frontpage .subnav-content {
```

Comment:
On the frontpage, the version selector was making the header jump around.
Here I fix that by using a `position: absolute` to remove the subnav content from the normal document flow.
Then I use `position: relative` on the subnav hover to anchor the subnav content to it.